docs/reference/ml/ml-shared.asciidoc

```diff
@@ -1276,7 +1276,11 @@ analytical processing. Once this limit is approached, data pruning becomes
 more aggressive. Upon exceeding this limit, new entities are not modeled. The
 default value for jobs created in version 6.1 and later is `1024mb`. If the
 `xpack.ml.max_model_memory_limit` setting has a value greater than `0` and less
-than `1024mb`, however, that value is used instead. The default value is
+than `1024mb`, however, that value is used instead. If
+`xpack.ml.max_model_memory_limit` is not set, but
+`xpack.ml.use_auto_machine_memory_percent` is set, then the default
+`model_memory_limit` will be set to the largest size that could be assigned in
+the cluster if this is smaller than than `1024mb`. The default value is
```

Comment:
Does this mean it will be set to the largest size that could be assigned in the cluster capped at 1024mb?

Reply:
Yes, I can change it to say that if you think it's clearer.